### ISSUE TITLE
Debug training divergence on both theano/tensorflow

### ISSUE THREAD

**Issue by tomowind** [garlicbulb-puzhuo/ggo#75]

**Comment by oakcreek** [comment]
Solution 1: try increase optimizer learning rate. See branch '20170304'.

**Comment by tomowind** [comment]
```
$ git diff master
diff --git a/scripts/optimizer.py b/scripts/optimizer.py
index 27e381c..33ea871 100644
--- a/scripts/optimizer.py
+++ b/scripts/optimizer.py
@@ -1,4 +1,4 @@
 from keras.optimizers import Adam, SGD

 sgd = SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True, clipnorm=1.0)
-adam = Adam(clipnorm=1.0)
+adam = Adam(lr=0.01, clipnorm=1.0)
```

**Comment by tomowind** [comment]
See #79
